web/client-api/src/main/java/io/deephaven/web/client/api/tree/enums/JsAggregationOperation.java

```diff
@@ -22,5 +24,9 @@ public class JsAggregationOperation {
             UNIQUE = "Unique",
             // Array operation isn't legal in all contexts, just omit it for now
             // ARRAY = "Array",
+            // These need some other parameter to function, not supported yet
+            // SORTED_FIRST="SortedFirst",
+            // SORTED_LAST="SortedLast",
+            // WSUM = "WeightedSum";
             SKIP = "Skip";
```

Comment:
You removed this aggregation, should it be removed from here as well? Web UI does reference it in one location, but that could be removed too... your call: https://github.com/deephaven/web-client-ui/blob/main/packages/iris-grid/src/sidebar/aggregations/AggregationUtils.ts#L48

Reply:
I assumed it wasnt safe/nice to outright remove it, though I will make sure we warn if it is used. I'll deprecated it here for later removal?